solr/core/src/java/org/apache/solr/handler/admin/StatusOp.java

```diff
@@ -41,14 +39,6 @@ public void execute(CoreAdminHandler.CallInfo it) throws Exception {
     String reconcile = params.get(CoreAdminParams.RECONCILE_THRESHOLD);
     boolean isIndexInfoNeeded = Boolean.parseBoolean(null == indexInfo ? "true" : indexInfo);
     NamedList<Object> status = new SimpleOrderedMap<>();
-    NamedList<Object> nodeStatus = new SimpleOrderedMap<>();
-    if (it.handler.coreContainer != null) {
-      Path solrDataHome = it.handler.coreContainer.getCoreRootDirectory();
-      File dataDir = solrDataHome.toFile();
-      nodeStatus.add("indexDiskSizeTotal", dataDir.getTotalSpace());
-      nodeStatus.add("indexDiskSizeAvailable", dataDir.getUsableSpace());
-    }
-    it.rsp.add("nodeStatus", nodeStatus);
```

Comment:
tbh I'd be fine leaving this, if you like. Either way.

This might even be upstreamable. Serves other directory impls, and gives data even when `indexInfo=false` (more efficient).